integ-test/build.gradle

```
@@ -180,6 +179,11 @@ integTest {
         }
     }
 
+    if(getOSFamilyType() != "linux" && getOSFamilyType()!="darwin") {
```

Is it possible to do
```suggestion
    if (getOSFamilyType() != "windows") {
```

---

Should be == windows. Probably a good thing to do otherwise integ tests would pass silently for any new instance types with out verifying datasource and prometheus tests.